### PR TITLE
Add manifest file for kubernetes

### PR DESCRIPTION
**Pull request by bgpat**
## WHY

Deploy to kubernetes cluster.

## WHAT

Add manifest files.

- Pods
  - server (gddo-server)
  - redis
- Services
  - server:80 (LoadBalancer)
  - redis:6379 (NodePort)

### CODE REVIEW

File: kubernetes/server.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  replicas: 1
+  template:
+    metadata:
+      name: server
+      labels:
+        name: server
+        role: web
+    spec:
+      imagePullSecrets:
+     - name: wantedlybotkey
```
Comment (koudaiii) [inline kubernetes/server.yaml]: yaml がそもそも正しくないかも

File: kubernetes/redis.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  replicas: 1
+  template:
+    metadata:
+      name: redis
+      labels:
+        name: redis
+        role: db
+    spec:
+      containers:
+      - image: redis
```
Comment (koudaiii) [inline kubernetes/redis.yaml]: version 指定したほうがいいかも

File: kubernetes/redis-svc.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis
+  namespace: godoc
+  labels:
+    name: redis
+    role: db
```
Comment (koudaiii) [inline kubernetes/redis-svc.yaml]: cache でも良いかもw

**Comment by bgpat** [comment]
Fixed. Please review.

**Comment by koudaiii** [comment]
LGTM!!

**Pull request merged**